plugins/inputs/kube_lite/node.go

```diff
@@ -0,0 +1,45 @@
+package kube_lite
+
+import (
+	"context"
+
+	"github.com/ericchiang/k8s/apis/core/v1"
+
+	"github.com/influxdata/telegraf"
+)
+
+var (
+	nodeMeasurement = "kube_node"
+)
+
+func registerNodeCollector(ctx context.Context, acc telegraf.Accumulator, ks *KubernetesState) {
```

Comment:
I think the name here is wrong because this doesn't register anything, maybe `collectNodes`